project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 // You may use this file to add plugin dependencies for sbt.
-resolvers += "Spark Packages repo" at "https://dl.bintray.com/spark-packages/maven/"
+resolvers += "Spark Packages repo" at "https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/"
```

Comment:
This should be changed to `https://repos.spark-packages.org/` instead. See: http://apache-spark-user-list.1001560.n3.nabble.com/Bintray-replacement-for-spark-packages-org-td40128.html

Reply:
I'm not sure if I have permissions to re-request a review, but I've made this change and this PR is ready for review again.